src/widgets/notebook/NotebookEditor.tsx

```diff
@@ -119,11 +120,12 @@ export function Editor({ widget }: { widget: INotebookWidget }) {
         TaskItem.configure({
           nested: true,
           onReadOnlyChecked: (node, checked) => {
-            if (widget.properties.allowReadOnlyCheck) {
+            if (widget.properties.allowReadOnlyCheck && enabled) {
```

Comment:
I guess when option is true the user should be able to check also as non administrator

Reply:
Currently only admins can check